applications/GeoMechanicsApplication/custom_workflows/adaptive_time_incrementor.cpp

```diff
@@ -78,6 +81,7 @@ void AdaptiveTimeIncrementor::PostTimeStepExecution(const TimeStepEndState& rRes
     else if (rResultantState.Converged() &&
              (rResultantState.num_of_iterations < mMinNumOfIterations)) {
         mDeltaTime *= mIncreaseFactor;
+        mDeltaTime = std::min(mDeltaTime,mMaxDeltaTime);
```

Comment:
We could also express this in a single statement:
```suggestion
        mDeltaTime = std::min(mDeltaTime * mIncreaseFactor, mMaxDeltaTime);
```

Reply:
Done